### PR TITLE
PR of Bradys PR

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

**Comment by halter73** [comment]
Is this safe to edit?

**Comment by Rick-Anderson** [comment]
Yes, I won't make a commit until you're finished.

**Comment by halter73** [comment]
Cool. I made my edits.

**Comment by Rick-Anderson** [comment]
@bradygaster let me know if I can merge this into your PR
@halter73  you can see my commit for changes I made:

Removed future tense and all.
Remove personal pronouns where it's as clear or more clear without them.

`wiring it up` Jargo that doesn't MT (machine translate)
`hooking` Jargo that doesn't MT

File: aspnetcore/release-notes/aspnetcore-3.0.md [hunk 1]
```diff
@@ -134,9 +134,9 @@ For more information, see <xref:blazor/index>.
 
 ## SignalR
 
-The SignalR JavaScript client has changed from being `@aspnet/signalr` to `@microsoft/signalr`. To react to this change, you will need to change your references in `package.json` files, require statements, and ECMAScript import statements. 
+See [Update SignalR code](xref:migration/22-to-30#update-signalr-code) for migration instructions.
 
-In the JavaScript and .NET Clients for SignalR, we've added support for automatic reconnection via the `withAutomaticReconnect()` method on the `HubConnectionBuilder`. By default, the client will try to reconnect immediately and after 2, 10, and 30 seconds. Enlisting in automatic reconnect is opt-in, but simple via this new method.
+In the JavaScript and .NET Clients for SignalR, support was added for automatic reconnection. By default, the client tries to reconnect immediately and retry after 2, 10, and 30 seconds if necessary. If the client successfully reconnects, it receives a new connection ID. Automatic reconnect is opt-in:
```
Comment (bradygaster) [inline aspnetcore/release-notes/aspnetcore-3.0.md]: *support was added* feels like passive voice. y'all used to beat me up for that, but if you like it, i'm cool with it. 👍

File: aspnetcore/release-notes/aspnetcore-3.0.md [hunk 2]
```diff
@@ -145,18 +145,28 @@ const connection = new signalR.HubConnectionBuilder()
     .build();
 ```
 
-By passing an array of millisecond-based durations to the method, you can be very granular about how your reconnection attempts occur over time.
+The reconnection intervals can be specified by passing an array of millisecond-based durations:
 
 ```javascript
 .withAutomaticReconnect([0, 3000, 5000, 10000, 15000, 30000])
-//.withAutomaticReconnect([0, 2000, 10000, 30000]) yields the default behavior
+//.withAutomaticReconnect([0, 2000, 10000, 30000]) The default intervals.
```
Comment (bradygaster) [inline aspnetcore/release-notes/aspnetcore-3.0.md]: 👍

**Pull request merged**